pyvista/plotting/render_window_interactor.py

```diff
@@ -912,6 +1016,32 @@ def _mouse_left_button_click(self, x=None, y=None, count=1):
             self._mouse_left_button_press(x, y)
             self._mouse_left_button_release()
 
+    def _mouse_middle_button_press(
+        self, x=None, y=None
+    ):  # pragma: no cover # numpydoc ignore=PR01,RT01
+        """Simulate a middle mouse button press.
+
+        If ``x`` and ``y`` are entered then simulates a movement to
+        that position.
+
+        """
+        if x is not None and y is not None:
+            self._mouse_move(x, y)
+        self.interactor.MiddleButtonPressEvent()
+
+    def _mouse_middle_button_release(
+        self, x=None, y=None
+    ):  # pragma: no cover # numpydoc ignore=PR01,RT01
+        """Simulate a middle mouse button release."""
+        if x is not None and y is not None:
+            self._mouse_move(x, y)
+        self.interactor.MiddleButtonReleaseEvent()
+
+    def _mouse_middle_button_click(self, x=None, y=None, count=1):
```

Comment:
This function is not needed for this code, and it is only used for testing purposes I think.  I propose that we don't need to have coverage for these lines at this time.